src/main/java/org/jabref/model/entry/FieldProperty.java

```diff
@@ -20,16 +17,13 @@
     EDITOR_TYPE,
     PAGINATION,
     TYPE,
-    CROSSREF,
```

Comment:
Why is crossref gone here?

Reply:
It was not used (only assigned) and there is `single/multiple entry link` which looks the proper generalization.